exporter/awsemfexporter/config.go

```diff
@@ -66,3 +73,51 @@ type Config struct {
 	// logger is the Logger used for writing error/warning logs
 	logger *zap.Logger
 }
+
+type MetricDescriptor struct {
```

Comment:
can we move the `MetricDescriptor` changes to a new file instead of in `config.go`?

Reply:
Wanted to move `MetricDescriptor` related handlings in one file and the other one `MetricDeclaration` done the same.